template.rb

```diff
@@ -29,11 +36,20 @@
 GITHUB_OAUTH_ID=#{github_id}
 GITHUB_OAUTH_SECRET=#{github_secret}
 GITHUB_API_TOKEN=#{github_token}
+PORT=3000
```

Comment:
Setting it to 3000 since thats how its configured in `secrets.yml`. Im thinking it makes sense to as the user which PORT they want to use for development, and default to 3000.